internal/config/releases.go

```diff
@@ -63,8 +63,9 @@ func getLatestReleaseNames(limit int) ([]adapter.Version, error) {
 }
 
 // GetLatestReleases fetches the latest releases from the traefik mesh repository
-func GetLatestReleases(releases uint) ([]*Release, error) {
-	releaseAPIURL := "https://api.github.com/repos/traefik/mesh/releases?per_page=" + fmt.Sprint(releases)
+func GetLatestReleases() ([]*Release, error) {
+	// Making the results to 10 to avoid fetching lot of releases (to avoid CWE-88)
+	const releaseAPIURL = "https://api.github.com/repos/traefik/mesh/releases?per_page=10"
```

Comment:
Makes sense. 

@meshery/adapter-maintainers please take note.

Reply:
This uses github API, I have added the page scraping way in meshkit. I think,  that should be reused everywhere. We need to get off github API completely for fetching latest releases

Reply:
Yes, very good point, @Revolyssup. 

@manav1403 or @piyushsingariya might point out examples of where this has been done elsewhere, so that we can move entirely away from any api.github.com requests and over to github.com requests.

Reply:
Never mind. I think @Revolyssup has this covered here - https://github.com/meshery/meshery/pull/4667/files/040346d6d219572aad6e1979b6cd0a00e0db083c..dd64b57cf7825d0094b4bc00a512f945fccfc942

Reply:
OK... This makes sense, if that change is implemented I'll include the method here too